frontend/src/components/core/Block/Block.tsx

```diff
@@ -169,6 +175,12 @@ const VerticalBlock = (props: BlockPropsWithoutWidth): ReactElement => {
         return (
           <StyledVerticalBlock width={width} data-testid="stVerticalBlock">
             <ChildRenderer {...propsWithNewWidth} />
+            {main && (
+              <StyledIFrameResizerAnchor
+                isEmbedded={embedded}
```

Comment:
Curious the use of the embedded. I see it in the styled components, but I wonder are you checking if it's in an iframe or in an embedded iframe?

My guess is that you don't need the "embedded" feature, but you make it look good in case it's in the embedded mode?

Reply:
Yes, I just made it so it works in both cases. Of course it makes a lot of sense in the embedded case (video demo of the description) but it can also be useful in the regular mode (Streamlit in Hugging Face Spaces for instance)